x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/utils/mock_providers.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FieldType, InferenceProvider } from '../types/types';
+
+export const mockProviders: InferenceProvider[] = [
+  {
+    service: 'hugging_face',
+    name: 'Hugging Face',
+    task_types: ['text_embedding', 'sparse_embedding'],
+    configurations: {
+      api_key: {
+        default_value: null,
+        description: `API Key for the provider you're connecting to.`,
+        label: 'API Key',
+        required: true,
+        sensitive: true,
+        updatable: true,
+        type: FieldType.STRING,
+        supported_task_types: [],
```

Comment:
Even in the mockup supported_task_types supposed to have the proper values like `supported_task_types: ['text_embedding', 'sparse_embedding'],` to literally describe for which taskType the field will be rendered in UI. In other case there will be a default value like always render